src/Playwright.NUnitTest/PlaywrightTest.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Concurrent;
+using System.Threading.Tasks;
+using System.Threading;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using Microsoft.Playwright;
+
+namespace Microsoft.Playwright.NUnitTest
+{
+    public class PlaywrightTest
+    {
+        private static Task<IPlaywright> playwrightTask_ = Microsoft.Playwright.Playwright.CreateAsync();
+
+        internal IPlaywright Playwright { get; private set; }
+
+        [SetUp]
```

Comment:
I seems we can have async `OneTimeSetup` https://docs.nunit.org/articles/nunit/writing-tests/attributes/onetimesetup.html

Reply:
Not really, we want it shared across tests.